en/rocky/8/guides/asterisk_installation.md

```diff
@@ -0,0 +1,77 @@
+# What's Asterisk
+Asterisk is an open source framework for building communications applications. Asterisk turns an ordinary computer into a communications server. Asterisk powers IP PBX systems, VoIP gateways, conference servers and other custom solutions. It is used by small businesses, large businesses, call centers, carriers and government agencies, worldwide. Asterisk is free and open source. Asterisk is sponsored by Sangoma
+
+# Prerequisite
+## Update RockyLinux
+
+    sudo dnf -y update
+
+## Set Hostname
+    sudo hostnamectl set-hostname asterisk.example.com
+
+## Add EPEL Repository
+    sudo dnf -y install epel-release
+    sudo dnf config-manager --set-enabled powertools
+    
+## Install Development Tools
+    sudo dnf group -y install "Development Tools"
+    sudo dnf -y install git wget  
+
+
+# Install Asterisk
+## Compile Asterisk
+    cd ~
+    wget http://downloads.asterisk.org/pub/telephony/asterisk/asterisk-18-current.tar.gz
+    tar xvfz asterisk-18-current.tar.gz
+    cd asterisk-18*/
+    contrib/scripts/install_prereq install
+    ./configure --libdir=/usr/lib64 --with-jansson-bundled=yes
+
+## Set Asterisk menu options [For more options]
+    make menuselect 
+
+## Build and Install Asterisk
+    make
+    make install
+    make basic-pbx 
+    make config
```

Comment:
Did you mean `sudo make install-configs`?

Also, you need to specify the source of the config files you are installing.

`sudo CONFIG_SRC=configs/samples/ make install-configs`

An explanation would be good as well.  Perhaps the user is fine with the basic PBX config files to test things out.